src/client/src/client_main.cpp

```diff
@@ -0,0 +1,59 @@
+//
+// Created by kevin on 6/25/22.
+//
+#include <ctime>
+#include <cstring>
+#include <ros/ros.h>
+#include <client/current_time.h>
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Found C system header after C++ system header. Should be: client_main.h, c system, c++ system, other.  [build/include_order] [4]